docs: Update CLI options

kachick opened the pull request:
Follow changes in #31 and #85

andrewbranch commented:
Thank you! Could you add a changeset per the comment above, since the README of the published package is what shows on npmjs.com?

kachick commented on packages/cli/README.md:
```diff
@@ -147,7 +147,6 @@ Specifies rules/problems to ignore (i.e. not raise an error for).
 
 The available values are:
 
-- `wildcard`
```
I can't find the diff in #31 and #85 for this option.

kachick commented:
Added the changeset and squashed commit again!

This is my first experience for using changeset. I didn't know the toolset.
Could you check again? 🙏

kachick commented:
Thank you!

And thank you for this project. This helped me a lot in understanding how to test package behavior in octokit/graphql-schema#925 🙏
Initially, I was fighting with `npm link`.
